tests/itest_specs/basic_round_trip/short_call.cairo

```diff
@@ -1 +1,1033 @@
+        let (opt_long_call_position_0) = ILiquidityPool.get_pools_option_position(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr,
+            option_side=0,
+            maturity=expiry,
+            strike_price=strike_price
+        );
+        assert opt_long_call_position_0 = 0;
+        let (opt_short_call_position_0) = ILiquidityPool.get_pools_option_position(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr,
+            option_side=1,
+            maturity=expiry,
+            strike_price=strike_price
+        );
+        assert opt_short_call_position_0 = 0;
+
+        // Test lpool_balance
+        let (call_pool_balance_0) = ILiquidityPool.get_lpool_balance(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr
+        );
+        assert call_pool_balance_0 = 11529215046068469760;
+        
+        let (put_pool_balance_0) = ILiquidityPool.get_lpool_balance(
+            contract_address=amm_addr,
+            lptoken_address=lpt_put_addr
+        );
+        assert put_pool_balance_0 = 11529215046068469760000;
+
+        // Test pool_locked_capital
+        let (call_pool_locked_capital_0) = ILiquidityPool.get_pool_locked_capital(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr
+        );
+        assert call_pool_locked_capital_0=0;
+        let (put_pool_locked_capital_0) = ILiquidityPool.get_pool_locked_capital(
+            contract_address=amm_addr,
+            lptoken_address=lpt_put_addr
+        );
+        assert put_pool_locked_capital_0=0;
+
+        // test value of pools position
+        let (pools_pos_val_call) = ILiquidityPool.get_value_of_pool_position(
+            contract_address = amm_addr,
+            lptoken_address = lpt_call_addr
+        );
+        assert pools_pos_val_call = 0;
+        
+        let (pools_pos_val_put) = ILiquidityPool.get_value_of_pool_position(
+            contract_address = amm_addr,
+            lptoken_address = lpt_put_addr
+        );
+        assert pools_pos_val_put = 0;
+
+        ///////////////////////////////////////////////////
+        // BUY THE CALL OPTION
+        ///////////////////////////////////////////////////
+
+        %{ stop_warp_1 = warp(1000000000 + 60*60*12, target_contract_address=ids.amm_addr) %}
+
+        let strike_price = Math64x61.fromFelt(1500);
+        let one = Math64x61.fromFelt(1);
+
+        let (premia: Math64x61_) = IAMM.trade_open(
+            contract_address=amm_addr,
+            option_type=0,
+            strike_price=strike_price,
+            maturity=expiry,
+            option_side=1,
+            option_size=one,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr
+        );
+
+        assert premia = 627445539966218; // approx 0.0002 ETH which may sound like way too different from the long premia, but the trade_vol here is 91.6 and the premia is correct
```

Comment:
that's because its trade size of 20% of the pool... from top of your head, does this correspond to the numbers in the notebook you did few weeks/months ago?

Reply:
ok